roles/tower/defaults/main.yml

```diff
@@ -1,4 +1,7 @@
 ---
+tower_multitenant: false
```

Comment:
I would rather set the default to `true`, since many use cases (maybe the majority?) would have multi-tenancy as a default feature and have more than one user in the system.

Regardless, we need to document what this does (if true, it makes the tower/awx pod run with `privileged: true` and `AWX_PROOT_ENABLED` is true, and false is vice-versa), and link those docs to the [Security](https://docs.ansible.com/ansible-tower/2.3.1/html/userguide/security.html) docs page.

I do wonder why the docs for this setting were removed in 2.6.x...

Reply:
I like what you mentioned earlier in #15:

> Or maybe provide a way to do both and just write up the docs that describe some of the benefits/risks of both?

Basically, default to the more secure deployment, and we can also add the `Add privileged SCC to service account` task maybe... but allow the option for those who choose to turn off that extra security layer by changing the default from `true` to `false`.

Reply:
Agreed. I'll get this documented. I went with false as that actually looks what it is set to by default right now in the configmap template. So that will be a change from how the operator runs by default today.